content/faq/what-about-esp8266-esp32.md

```diff
@@ -7,7 +7,7 @@ As of February 2019 there is now an official project from Espressif to add the X
 
 For the discussion of this see [this forum thread](https://www.esp32.com/viewtopic.php?t=9226).
 
-The repository that contains the Xtensa fork of LLVM is located at [https://github.com/espressif/llvm-xtensa](https://github.com/espressif/llvm-xtensa)
+The repository that contains the Xtensa fork of LLVM is located at [https://github.com/espressif/llvm-project](https://github.com/espressif/llvm-project)
 
 The repository that contains the Xtensa fork of Clang is located at [https://github.com/espressif/clang-xtensa](https://github.com/espressif/clang-xtensa).
```

Comment:
This line should be removed: Clang is now in the same repository as the entire LLVM project.